packages/paste-design-tokens/tokens/global/background-color.yml

```diff
@@ -32,6 +32,9 @@ props:
   color-background-brand-highlight:
     value: "{!amaranth}"
     comment: Highlight brand background, accessible with black text
+  color-background-brand-highlight-lightest:
+    value: "{!amaranth-transparent-10}"
+    comment: Lightest highlight brand background, accessible with black text
```

Comment:
Just a note for future reference: Depending on what ends up being easier to implement for the red-to-white gradient on the homepage (https://www.figma.com/file/I6ZWKfR6ftC9n99icmdh6V/Homepage-Brainstorm?node-id=541%3A142). This could either stay as is, or it could be red-10 (and we could remove the new alias).